pkgs/top-level/all-packages.nix

```diff
@@ -24016,7 +24016,7 @@ in
 
   vdirsyncer = with python3Packages; toPythonApplication vdirsyncer;
 
-  vdirsyncerStable = with python3Packages; toPythonApplication vdirsyncerStable;
+  vdirsyncerStable  = throw "vdirsyncerStable is not supported anymore. Use vdirsyncer instead. For more info, see: https://github.com/NixOS/nixpkgs/issues/103026#issuecomment-723428168";
```

Comment:
this should be moved into `pkgs/top-level/aliases.nix`

Reply:
Done.